blockchain/state/sync_test.go

```diff
@@ -13,70 +13,86 @@
 //
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+//
+// This file is derived from core/state/sync_test.go (2020/05/20).
+// Modified and improved for the klaytn development.
 
 package state
 
 import (
 	"bytes"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/crypto"
+	"github.com/klaytn/klaytn/storage/database"
+	"github.com/klaytn/klaytn/storage/statedb"
 	"math/big"
 	"testing"
-
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core/rawdb"
-	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/ethdb"
-	"github.com/ethereum/go-ethereum/ethdb/memorydb"
-	"github.com/ethereum/go-ethereum/trie"
 )
 
 // testAccount is the data associated with an account used by the state tests.
 type testAccount struct {
-	address common.Address
-	balance *big.Int
-	nonce   uint64
-	code    []byte
+	address    common.Address
+	balance    *big.Int
+	nonce      uint64
+	code       []byte
+	storageMap map[common.Hash]common.Hash
```

Comment:
Klaytn accounts have more fields; humanReadable and key.

Reply:
I will consider to add more test case for our Klaytn account laster. :)